TicketBookingBackend/TicketBookingAPI/TicketBookingAPI/Extensions/RegisterDependenciesExtension.cs

```diff
@@ -16,6 +16,11 @@ public static void AddDependencies(this WebApplicationBuilder builder)
             builder.Services.AddScoped<ITicketRepository, TicketRepository>();
             builder.Services.AddScoped<IUserService, UserService>();
             builder.Services.AddScoped<IUserRepository, UserRepository>();
+
+            builder.Services.AddScoped<TicketBooking.Services.Interfaces.IBusService,TicketBooking.Services.Classes.BusService>();
```

Comment:
Lets create one more extension methods to segregate Service and Repository in a separate class file